docs/source/kernel-library.md

```diff
@@ -0,0 +1,22 @@
+## Standalone Remote Kernel Execution 
+#####a.k.a "Library Mode"
```

Comment:
Shoot - just noticed this needs a space between the heading and `a.k.a`.

Reply:
Ugh my bad...

I blame pycharm.

This is fixed.